src/index.ts

```diff
@@ -43,18 +42,17 @@ class ApolloError extends ExtendableError {
       name,
       time_thrown,
       data,
+      ...(_showLocations ? {
       path,
       locations
+      } : {} )
     };
-    if (_showLocations) {
-      error.locations = locations;
-      error.path = path;
-    }
+
     return error;
   }
 }
 
-export const isInstance = e => e instanceof ApolloError;
+export const isApolloError = e => e instanceof ApolloError;
```

Comment:
I'm not sure how I feel about making a breaking change.  I'd be okay with exporting this under two different names for backwards compatibility.

Reply:
That would work, something like:
```
export const isApolloError = e => e instanceof ApolloError;
export const isInstance = isApolloError // deprecated, use isApolloError instead
```

Reply:
Yeah that's fine.